src/shogun/evaluation/CrossValidationStorage.cpp

```diff
@@ -187,23 +97,32 @@ operator==(const CrossValidationFoldStorage& rhs) const
 	       m_evaluation_result == rhs.m_evaluation_result;
 }
 
+EEvaluationResultType CrossValidationFoldStorage::get_result_type() const {
+	return CROSSVALIDATIONFOLDSTORAGE_RESULT;
+}
+
+void CrossValidationFoldStorage::print_result() {
+
+}
+
 /** CrossValidationStorage **/
 
-CrossValidationStorage::CrossValidationStorage() : CSGObject()
+CrossValidationStorage::CrossValidationStorage() : CEvaluationResult()
 {
 	m_num_runs = 0;
 	m_num_folds = 0;
 	m_expose_labels = NULL;
```

Comment:
I see the name was my idea :see_no_evil: maybe just change it :)

Reply:
I do like it! :D I would keep it as it is